fantasycourt.nl/content/praktische_info/faq/_index.nl.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Vragen en Antwoorden
+og:
+  description: "Fantasy Court - Een gratis fantasy festival in Eindhoven"
+
+---
+
+
+
+{{< message >}}
+_Hier is een antwoord voor vele vragen_
+_Dezen kunnen uw bezoek behagen_
+{{< /message >}}
+
+#  Vragen en Antwoorden
+Dit zijn veel gestelde vragen en de antwoorden daarop. Heb je een vraag die hier niet tussen staan? Voel je vrij om [een mail](mailto:organisatie@fantasycourt.nl?subject=Vraag)  te sturen. 
+
+## Algemeen
+### Kan ik geld bijpinnen op Fantasy Court?
+Fantasy court heeft geen geldopnamepunt op locatie. Veel verkopers accepteren zowel contant als pinbetalingen. Maar sommigen kunnen enkel één van beiden accepteren. Zorg alstublieft dat je hierop voorbereid bent. Het dichtstbijzijnde pinautomaat is in de kruisstraat op ruwweg een kilometer lopen. 
+
+## Wat voor soort kramen zullen er zijn?
+We hebben een grote verscheidenheid aan standhouders. Je kan een lijst vinden op de [programmapagina]({{< ref "/programma_2024" >}}) zodra ze bevestigd zijn.
+
+
+## Muziek
+### Hebben jullie muziek?
```

Comment:
Misschien is het tijd om (gedeeltelijk) afscheid te nemen van dit lettertype. Het maakt deze pagina heel lastig leesbaar. Drie `###` vertaalt naar een `<h3>`. Je kan daar een andere styling voor maken